plugins/jenkins/src/components/JobRunsTable/JobRunsTable.tsx

```diff
@@ -0,0 +1,186 @@
+          </Typography>
+        </Box>
+      );
+    },
+  },
+  {
+    title: 'Result',
+    field: 'result',
+    render: (row: Partial<JobBuild>) => {
+      return (
+        <Box display="flex" alignItems="center">
+          {row.inProgress ? (
+            <Typography>In Progress</Typography>
+          ) : (
+            <JenkinsRunStatus status={row?.result} />
+          )}
+        </Box>
+      );
+    },
+  },
+  {
+    title: 'Duration',
+    field: 'duration',
+    render: (row: Partial<JobBuild>) => {
+      return (
+        <Box display="flex" alignItems="center">
+          <Typography>
+            {row?.duration
+              ? (row.duration / 1000).toFixed(1).toString().concat(' s')
+              : ''}
+          </Typography>
+        </Box>
+      );
+    },
+  },
+
+  {
+    title: 'Actions',
+    render: (row: Partial<JobBuild>) => {
+      const ActionWrapper = () => {
+        return (
+          <div style={{ width: '98px' }}>
+            {row?.url && (
+              <Tooltip title="View build">
+                <IconButton href={row.url} target="_blank">
+                  <VisibilityIcon />
+                </IconButton>
+              </Tooltip>
+            )}
+          </div>
+        );
+      };
+      return <ActionWrapper />;
+    },
+    width: '10%',
+  },
+];
+
+type Props = {
+  loading: boolean;
+  jobRuns?: Job;
+  page: number;
+  onChangePage: (page: number) => void;
+  total: number;
+  pageSize: number;
+  onChangePageSize: (pageSize: number) => void;
+};
+
+export const JobRunsTableView = ({
+  loading,
+  pageSize,
+  page,
+  jobRuns,
+  onChangePage,
+  onChangePageSize,
+  total,
+}: Props) => {
+  const builds = jobRuns?.builds.slice(
+    page * pageSize,
+    page * pageSize + pageSize,
+  );
+  let sumOfAllSuccessfullJobDuration = 0;
+
+  const successfulJobCount =
+    builds?.reduce((count, build) => {
+      if (!build.inProgress && build.result === 'SUCCESS') {
+        sumOfAllSuccessfullJobDuration += build.duration;
+        return count + 1;
+      }
+      return count;
+    }, 0) || 0;
+
+  let avgTime;
+
+  if (successfulJobCount > 0) {
+    avgTime = (sumOfAllSuccessfullJobDuration / successfulJobCount / 1000)
+      .toFixed(1)
+      .toString();
+  }
+
+  return (
+    <Table
+      isLoading={loading}
+      options={{ paging: true, pageSize, padding: 'dense' }}
+      totalCount={total}
+      page={page}
+      data={builds ?? []}
+      onPageChange={onChangePage}
+      onRowsPerPageChange={onChangePageSize}
+      title={
+        <Box>
+          <Box display="flex" alignItems="center">
+            <img src={JenkinsLogo} alt="Jenkins logo" height="50px" />
+            <Box mr={2} />
+            <Typography variant="h6">Job Runs</Typography>
+          </Box>
+          <Box display="flex" alignItems="center" mt={2}>
+            <Typography variant="h6">
+              Average Build Time For Last {successfulJobCount} Successfull jobs
```

Comment:
Typo here on the text output still (not the var replacement)

Reply:
i have fixed this, thank you for your support, feedback and valuable time. @adamdmharvey